rpc/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-description = "Parity JSON-RPC servers."
+description = "Parity Ethereum JSON-RPC Servers"
```

Comment:
The plural in "servers" always struck me as odd. It's one, isn't it? @tomusdrw ?

Reply:
We do have multiple transports that can be considered multiple servers: WS, HTTP, IPC.

Reply:
I've addressed this in commit e9e39204302a31b7263e353dc78a4b7267663f3b